Configuration.sln

```diff
@@ -41,6 +41,10 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Configu
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Configuration.Binder.Test", "test\Microsoft.Framework.Configuration.Binder.Test\Microsoft.Framework.Configuration.Binder.Test.xproj", "{AE6FFE9B-6378-4D57-AA24-7D257F18B235}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.Framework.Configuration.File", "src\Microsoft.Framework.Configuration.File\Microsoft.Framework.Configuration.File.xproj", "{881E7CBC-492C-47C5-98A6-61DD1C753EE6}"
```

Comment:
I am liking `Microsoft.Framework.Configuration.FileExtensions` more and more for reasons explained in other comments. @Eilon would you be ok with that?